public/expressions/overlay_anomalies.ts

```diff
@@ -0,0 +1,193 @@
+// /*
+//  * SPDX-License-Identifier: Apache-2.0
+//  *
+//  * The OpenSearch Contributors require contributions made to
+//  * this file be licensed under the Apache-2.0 license or a
+//  * compatible open source license.
+//  *
+//  * Any modifications Copyright OpenSearch Contributors. See
+//  * GitHub history for details.
+//  */
```

Comment:
For new files we can use the short form:
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
will change